bin/node/miden-node.toml

```diff
@@ -19,3 +19,4 @@ store_url = "http://localhost:28943"
 endpoint = { host = "localhost", port = 28943 }
 database_filepath = "miden-store.sqlite3"
 genesis_filepath = "genesis.dat"
+blocks_dir = "blocks"
```

Comment:
Similar to one of the previous comments: I would change "blocks" to "blockstore".